---
name: Martin HÃ¶lzer
member: martin-hoelzer
image: images/members/martin-hoelzer.jpg
role: team-lead
description: Team Lead/Group PI
aliases:
  - Martin Hoelzer
  - M. Hoelzer
links:
  orcid: 0000-0001-7090-8717
  google-scholar: DMZ7Hc8AAAAJ
  github: hoelzer
  twitter: martinhoelzer
---

I'm Martin, a passionate bioinformatician with a rich background in sequencing data analysis, currently leading the "Research & Innovation" team within the [Genome Competence Center (MF1)](https://www.rki.de/EN/Content/Institute/DepartmentsUnits/MF/MF1/mf1_node.html) at the Robert Koch Institute (RKI) in Berlin, Germany. I earned my PhD working on "The Dark Art of Next-Generation Sequencing: fundamental approaches for genomics, transcriptomics, and differential gene expression" at the University of Jena in 2018 ([thesis](https://suche.thulb.uni-jena.de/Record/1013860616)). During my postdoc, I worked on real-time long-read sequencing and its application in metagenomics and virus detection, e.g. in groundwater ([CRC AquaDiva](https://www.aquadiva.uni-jena.de/)). I continued to deepend my knowledge of nanopore metagenomics and pathogen detection as a visiting scientist in the [Microbiome Informatics group](https://www.ebi.ac.uk/about/teams/microbiome-informatics) at the European Bioinformatics Institute (EBI) in Hinxton, UK. 

In 2020, I moved to the RKI, where I previously worked as deputy head of the Bioinformatics unit, and conducted research in the areas of real-time nanopore sequencing, pipeline development, and genomic surveillance of SARS-CoV-2. With the establishment of the [Genome Competence Center (MF1)](https://www.rki.de/EN/Content/Institute/DepartmentsUnits/MF/MF1/mf1_node.html) in March 2023, I've taken the helm of the "Research & Innovation" team, guiding our interdisciplinary group in leveraging technology platforms and data analysis to forge new paths in experimental and bioinformatics approaches for cutting-edge research and applications.

<!--My research interests are vast, delving into transcriptomics & comparative genomics, _de novo_ assembly & annotation, and metagenomics & metatranscriptomics. A strong advocate for automation, I'm dedicated to creating user-friendly tools that facilitate researchers in getting biological insights, with a keen focus on workflow management systems and containerization to enhance our capabilities.

My team aims to bridge the gap between life sciences and computer science, employing advanced sequencing technologies to analyze (meta-)genomic and transcriptomic datasets related to infectious diseases, immunology, and medical sciences. This interdisciplinary approach has been instrumental in advancing our understanding of pathogen-host interactions and in supporting pandemic surveillance and containment efforts.-->

---
